### DISCUSSION TITLE
Reason for requiring the old places api permission?

### DISCUSSION THREAD

**Discussion by dnmeyer06** [visgl/react-google-maps#447] [Q&A] [answered]
Hey all,

I'm using the package for autocomplete, and was getting permissions errors until I added the old `Places API` in addition the `Places API (New)` to my google maps key.

 What does the package use the old places API for?

Thanks

**Comment by usefulthink** [comment, marked as answer]
It seems that Google is in the middle of a migration from an older API architecture to a newer one, which requires the new scope to be added for some features. They probably don't want to (or haven't yet completed) port all the functionality of the old services over to the new one, so for now there are two of them. 

That's just how it is, and it has nothing to do with our library. 

_(Note: I'm not a googler and don't have insights into their development, this is all just observations and educated guessing)_
